libfreerdp/core/gateway/rdg.c

```diff
@@ -1427,7 +1428,7 @@ static BOOL rdg_process_keep_alive_packet(rdpRdg* rdg)
 
 static BOOL rdg_process_service_message(rdpRdg* rdg, wStream* s)
 {
-	const WCHAR* msg;
+	WCHAR* msg;
```

Comment:
Would have changed the function signature to `static BOOL rdg_read_http_unicode_string(wStream* s, const WCHAR** string, UINT16* lengthInBytes)` rather than removing the `const` here